samples/msal-react-sample/src/msal-react/MsalContext.ts

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { AuthenticationParameters, TokenResponse, TokenRenewParameters, Account,  } from "@azure/msal-browser";
+
+// TODO: Move this to msal-browser
+export interface IPublicClientApplication {
```

Comment:
This now exists in msal-browser, FYI.

Reply:
Thanks! After referencing the proper interface I see that the `@azure/msal-browser` package has made a lot of changes. I updated the code to work with the latest package version, but I think there's a lot of design decisions that are raised. `msal-browser` takes some key API changes that should probably modify how the `msal-react` component is expected to behave.